README.md

```diff
@@ -4,216 +4,311 @@
+or method definitions, and more.
 
-**Try it now:** [https://semgrep.live](https://semgrep.live/)
+## Quickstart
+
+After [installing semgrep](#installation), run the default set of checks with:
+
+```bash
+$ semgrep --config=r2c path/to/repo
+```
+
+Or try Semgrep in your browser: [https://semgrep.live](https://semgrep.live/).
 
 ## Overview
 
-Language support:
+### Key features
+
+* **Language aware**: Semgrep parses source code into Abstract Syntax Trees (ASTs), so it understands function calls, method and class definitions, conditionals, and more. 
+  * Unlike regexes, Semgrep won't get tripped up matching things you don't care about in comments or string literals.
+* **Fast**: Speedy enough to run on every build, commit, or file save.
+* **Batteries included**: Leverage hundreds of preexisting security checks for popular languages and frameworks in the community [registry](https://semgrep.live/r), including coverage for the [OWASP Top 10](https://owasp.org/www-project-top-ten/).
+* **Handles multiple languages**: No need to learn and maintain multiple tools for your polyglot environment (e.g. ESLint, find-sec-bugs, rubocop, gosec, ...).
+* **Easy to tweak**: Rules look like the code you’re searching, no static analysis PhD required.
+* Does **not** require compilation, Semgrep runs on source code directly. 
+* **Easy to integrate**: Highly portable and many CI and git hook integrations already exist. Run via CLI or Docker, output `--json` and pipe results into whatever system you want.
+
+
+### Language support
+
+| **Python** | **Javascript** | **Go &nbsp; &nbsp; &nbsp;** | **Java &nbsp;** | **C &nbsp; &nbsp; &nbsp; &nbsp;** | **Ruby** | **Typescript** | **PHP &nbsp; &nbsp;** |
+|:-----------|:---------------|:----------------------------|:----------------|:----------------------------------|:---------|:---------------|:----------------------|
+| ✅          | ✅              | ✅                           | ✅               | ✅                                 | 🚧 |  Coming...      | Coming...             |
+
+... and many more coming.
+
+### Pattern Syntax Teaser
 
-| **Python** | **Javascript** | **Go &nbsp; &nbsp; &nbsp;** | **Java &nbsp;** | **C &nbsp; &nbsp; &nbsp; &nbsp;** | **Typescript** | **PHP &nbsp; &nbsp;** |
-|:-----------|:---------------|:----------------------------|:----------------|:----------------------------------|:---------------|:----------------------|
-| ✅          | ✅              | ✅                           | ✅               | ✅                                 | Coming...      | Coming...             |
+One of the most unique and useful things about Semgrep is how easy it is to write and iterate on queries. 
 
-Example patterns:
+The goal is to make it as *easy as possible* to go from an idea in your head to finding the code patterns you intend to.
 
-| **Pattern**                                                        | **Matches**                                                |
```

Comment:
@clintgibler could you comment on why you removed this? this is actually the only place we have "pattern X matches code Y" and I would miss it if it was gone -- it's a nice easy way to grok the syntax deductively.

Reply:
@ievans Agreed, I like having a brief table of "pattern X matches Y code." 

To me, maybe it was some sort of combination of the patterns being links, the denseness of the table, maybe the patterns themselves - it felt a bit cluttered and hard to read / unintuitive. I felt like I had to focus to understand what each row was trying to convey, and that it wasn't obvious.

I'm happy to put this back as is, but at some point I'd like to either change the formatting, the code snippets, or somehow make the examples pop more / be clearer.

Reply:
From a user:

> `$X == $X` is not narratively very poignant to me. It makes it looks like it’s just for code patterns, not variant analysis